multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -24,20 +31,672 @@ use node_lib::database::db_initializer::{
+        + owed_to_serv_node_2_minor)
+        - gwei_to_wei::<u128, u64>(2_345_678);
+    let agreed_transaction_fee_unit_price_major = 60;
+    let transaction_fee_needed_to_pay_for_one_payment_major = {
+        // We will need less but this should be accurate enough
+        let txn_data_with_maximized_cost = [0xff; 68];
+        let gas_limit_dev_chain = {
+            let const_part = web3_gas_limit_const_part(Chain::Dev);
+            u64::try_from(compute_gas_limit(
+                const_part,
+                txn_data_with_maximized_cost.as_slice(),
+            ))
+            .unwrap()
+        };
+        let transaction_fee_margin = Percentage::new(15);
+        transaction_fee_margin
+            .add_percent_to(gas_limit_dev_chain * agreed_transaction_fee_unit_price_major)
+    };
+    eprintln!(
+        "Computed transaction fee: {}",
+        transaction_fee_needed_to_pay_for_one_payment_major
+    );
+    let cons_node_transaction_fee_balance_minor =
+        2 * gwei_to_wei::<u128, u64>(transaction_fee_needed_to_pay_for_one_payment_major);
+    let test_global_config = TestInputsOutputsConfig {
+        ui_ports_opt: Some(Ports {
+            consuming_node: consuming_node_ui_port,
+            serving_node_1: serving_node_1_ui_port,
+            serving_node_2: serving_node_2_ui_port,
+            serving_node_3: serving_node_3_ui_port,
+        }),
+        // Should be enough only for two payments, the least significant one will fall out
+        cons_node_initial_transaction_fee_balance_minor_opt: Some(
+            cons_node_transaction_fee_balance_minor + 1,
+        ),
+        cons_node_initial_service_fee_balance_minor,
+        debts_config: Either::Right(FullySpecifiedSimulatedDebts {
+            // This account will be the most significant and will deserve the full balance
+            owed_to_serving_node_1: AccountedDebt {
+                balance_minor: owed_to_serv_node_1_minor,
+                age_s: payment_thresholds.maturity_threshold_sec + 1000,
+            },
+            // This balance is of a middle size it will be reduced as there won't be enough
+            // after the first one is filled up.
+            owed_to_serving_node_2: AccountedDebt {
+                balance_minor: owed_to_serv_node_2_minor,
+                age_s: payment_thresholds.maturity_threshold_sec + 100_000,
+            },
+            // This account will be the least significant and therefore eliminated
+            owed_to_serving_node_3: AccountedDebt {
+                balance_minor: owed_to_serv_node_3_minor,
+                age_s: payment_thresholds.maturity_threshold_sec + 30_000,
+            },
+        }),
+        payment_thresholds_all_nodes: payment_thresholds,
+        cons_node_transaction_fee_agreed_unit_price_opt: Some(
+            agreed_transaction_fee_unit_price_major,
+        ),
+        // It seems like the ganache server sucked up quite less than those 55_000 units of gas??
```

Comment:
Maybe you'd prefer to improve the comment.

```
The remainder balance expected by ganache after txns were successful
```

Maybe something different according to your understanding.